test/e2e_node/eviction_test.go

```diff
@@ -500,15 +501,38 @@ var _ = SIGDescribe("PriorityPidEvictionOrdering [Slow] [Serial] [Disruptive][No
 		specs[2].pod.Spec.PriorityClassName = highPriorityClassName
 		runEvictionTest(f, pressureTimeout, expectedNodeCondition, expectedStarvedResource, logPidMetrics, specs)
 	})
+
+	ginkgo.Context(fmt.Sprintf(testContextFmt, expectedNodeCondition)+"; PodDisruptionConditions enabled [Feature:PodDisruptionConditions]", func() {
```

Comment:
The whole eviction test is currently failing, so I suggest we move this out into a separate PR since we have no way of testing this

https://github.com/kubernetes/kubernetes/issues/107063

cc @CoderSherlock who's been looking into eviction tests

Reply:
I've tested it locally but indeed I could not find any pull configuration to test it. Do you think if we move it into a separate PR can this be then decoupled from the KEP? It might be hard to get it merged before the Test Freeze if it depends on https://github.com/kubernetes/kubernetes/issues/107063.

Reply:
Opened a PR against test-infra: https://github.com/kubernetes/test-infra/pull/27897. Can I run the tests before the test-infra PR merges into master?